### DISCUSSION TITLE
双拼键盘方案

### DISCUSSION THREAD

**Discussion by wzgcn** [Mintimate/oh-my-rime#124] [Q&A]
能否做一个小鹤双拼的键盘方案？类似下面这个图示。
<img width="400" alt="双拼键盘" src="https://github.com/user-attachments/assets/ffce6053-5bd6-4be7-96b5-e70df7213a13">
好像有人做了五笔字根的键盘方案，如下图。
![五笔字根键盘](https://github.com/user-attachments/assets/e0a2378a-7d03-44ac-9343-77d1b9941061)

**Comment by YummyCocoa** [comment]
看图片，这个应该Android的同文输入法了。目前薄荷没有余力制作了，加上上次制作皮肤后，被仓官方群的群友谴责，薄荷应该不会再制作皮肤。

不过，按道理小鹤应该是有的，可以在其他地方再找找，或者基于别人的五笔改为小鹤的。
